### DISCUSSION TITLE
🔗 make Icons of floating buttons customizable (Duplicate of #28741)

### DISCUSSION THREAD

**Discussion by carlojpf** [elementor/elementor#29213] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

it will make the floating buttons applicable to non social media or contact pages like review pages and other custom features that many websites need

### Describe the solution you'd like.

Just like any icons in Elementor make the icons of the floating buttons changeable

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by highheelcreative** [comment]
This would also be good if we could include display conditions that include time of day 😀

**Comment by nicholaszein** [comment]
Hello there,

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #28741

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
